.github/renovate.json5

```diff
@@ -40,7 +40,7 @@
       "matchStrings": [
         "EARTHLY_VERSION: '(?<currentValue>.*?)'\\n"
       ],
-      "datasourceTemplate": "github-tags",
```

Comment:
this is intended for other managers besides just codeql?

Reply:
Yeah, I figured since we always download release artifact URLs it's better to use the existence of a release as the datasource everywhere.